nephio-workload-cluster/pv-cluster.yaml

```diff
@@ -16,6 +16,10 @@ spec:
   injectors:
   - kind: WorkloadCluster
```

Comment:
Trying to map from the command output of `kubectl explain PackageVariant.spec.injectors` and what you have in L17 and L22. I am getting the impression that `kind` and `name` are repeated twice, I could be reading this wrong
```
ubuntu@nephio-r1-e2e-vish:~$ kubectl explain PackageVariant.spec.injectors
KIND:     PackageVariant
VERSION:  config.porch.kpt.dev/v1alpha1

RESOURCE: injectors <[]Object>

DESCRIPTION:

     InjectionSelector specifies how to select in-cluster objects for resolving
     injection points.

FIELDS:
   group	<string>

   kind	<string>

   name	<string> -required-

   version	<string>

```